java-frontend/src/main/java/org/sonar/java/se/ExplodedGraph.java

```diff
@@ -70,7 +69,7 @@ public ProgramPoint(CFG.Block block, int i) {
 
     @Override
     public int hashCode() {
-      if(hashcode == 0) {
+      if (hashcode == 0) {
```

Comment:
Sorry for this, I will try to separate code reformatting from actual changes. I considered this minor enough, so I didn't bother splitting it. Not sure if you are aware, but you can add `?w=1` to the URL to make diff ignore whitespace (see here https://github.com/blog/967-github-secrets )